### DISCUSSION TITLE
[Discussion] Distrobox on a nested X Server (Xephyr)

### DISCUSSION THREAD

**Discussion by k1mbl3** [89luca89/distrobox#1898] [Q&A]
Is distrobox capable of running an entire window manager in a nested x server, like [x11docker](https://github.com/mviereck/x11docker) is capable of doing with Xepyhr or weston for GPU support? in my tests the window manangers always crashed due to the display already been in use.

**Comment by 89luca89** [comment]
I was able to do it, just need to explicitly set the display to another one

**Comment by k1mbl3** [comment]
> 

Could you prepare a little tutorial to replicate that? in my tests changing the display also didn't work
